ts/src/header-validator/trigger.test.ts

```diff
@@ -28,12 +28,12 @@ const testCases: jsontest.TestCase<Trigger>[] = [
         "not_filters": {"c": ["d"]},
         "source_keys": ["x"]
       }],
-      "aggregatable_values": {"e": 5},
+      "aggregatable_values": {"x": 5},
```

Comment:
Just curious, why are we changing this?

Reply:
So that `"x"` doesn't appear in two different locations, which could be confusing.